src/lib/components/blog/labels.svelte

```diff
@@ -34,15 +35,18 @@
     font-size: 1em;
 
     a {
-      background-color: var(--acm-light);
-      border-radius: 5px;
-      padding: 0.3em;
+      background-color: #eeeeee;
```

Comment:
suggest going back to acm-light or using acm-canvas here, avoid having "magic numbers" or mix of different off white throughout the codebase whenever possible

Reply:
also makes it dark mode reactive

Reply:
Got it.